src/assets/fonts/inter.css

```diff
@@ -0,0 +1,39 @@
+/* inter-regular - latin */
+@font-face {
```

Comment:
instead of using this approach, you should take a look into next/fonts

Reply:
Is there any benefit from using "next/fonts"?

Reply:
next/fonts will automatically optimize your fonts (including custom fonts) and remove external network requests for improved privacy and performance by self-hosting Google fonts. (see handout react styled components 1)